admin/app/templates/views/usage-with-letters.html

```diff
@@ -30,15 +30,13 @@ <h2 class='heading-small'>Text messages</h2>
             {{ big_number(sms_sent, 'sent', smaller=True) }}
             {{ big_number(sms_free_allowance, 'free allowance', smaller=True) }}
             {{ big_number(sms_allowance_remaining, 'free allowance remaining', smaller=True) }}
-            {#
               {% if sms_chargeable %}
                 {{ big_number(
                   sms_chargeable,
-                  'at {:.2f} cents per message'.format(sms_rate * 100),
+                  'at {:.3g}cents per message (excl. GST)'.format(sms_rate * 100),
```

Comment:
@acodegirl per https://guides.service.gov.au/content-guide/numbers-measurements/#measurement-and-weights “Abbreviate measures when using a number. Do not put a space between number and measure.” did you mean `c` instead of `cents`?

Reply:
replaced cents with c